Client/src/Components/Inputs/Radio/index.jsx

```diff
@@ -42,6 +42,9 @@ const Radio = (props) => {
 						width: 16,
 						height: 16,
 						boxShadow: `inset 0 0 0 1px ${theme.palette.secondary.main}`,
+						"&:not(.Mui-checked)": {
+							boxShadow: `inset 0 0 0 1px ${theme.palette.text.primary}70`, // Use theme text color for the outline
```

Comment:
Sorry for the delay, guys. I'm not sure about this, but it seems to have hot fixed the problem